src/go/rpk/pkg/cli/cmd/redpanda/tune/tune.go

```diff
@@ -41,22 +39,22 @@ type result struct {
 	errMsg    string
 }
 
-func NewTuneCommand(fs afero.Fs) *cobra.Command {
+func NewCommand(fs afero.Fs) *cobra.Command {
 	tunerParams := factory.TunerParams{}
 	var (
 		configFile        string
 		outTuneScriptFile string
 		cpuSet            string
 		timeout           time.Duration
 	)
-	baseMsg := "Sets the OS parameters to tune system performance." +
-		" Available tuners: all, " +
-		strings.Join(factory.AvailableTuners(), ", ")
+	baseMsg := "Sets the OS parameters to tune system performance"
+	longMsg := baseMsg + ".\n\n Available tuners:\n  -  all\n  -  " +
```

Comment:
This may look clearer with a formatted string and raw text,
```go
longMsg := fmt.Sprintf(`Sets the OS parameters to tune system performance.

Available tuners:

  - all
  %s

To learn more about a tuner, run 'rpk redpanda tune help <tuner name>'.
`, strings.Join(factory.AvailableTuners(), "\n. - "))
```
(also, missing trailing period).

Also, why not do these inline in the cobra command initialization, similar to how other commands do this?